unicodetools/src/main/java/org/unicode/draft/ScriptCount.java

```diff
@@ -83,7 +83,7 @@ public int compareTo(SecondaryInfo arg0) {
     }
 
     static class SecondaryCounts {
-        private final UCA uca = UCA.buildCollator(null);
+        private final UCA uca = UCA.buildDucetCollator();
```

Comment:
FYI: The null was for the Remap object which I removed. null meant DUCET, not CLDR.

The internal function now takes two primary weights, which could be -1 for the DUCET. Rather than make several call sites even less readable, I created a function that says "DUCET" and does not take parameters.